apollo-router/src/compute_job/mod.rs

```diff
@@ -129,6 +136,8 @@ impl From<ComputeJobType> for opentelemetry::Value {
 }
 
 pub(crate) struct Job {
+    subscriber: Dispatch,
+    span: Span,
```

Comment:
NIT: should we call it `parent_span` to make it more explicit ?

Reply:
[a5ac86a](https://github.com/apollographql/router/pull/7236/commits/a5ac86ad11bda3a3ae377c63bd872b531566d80b)